v3/how-to-guides/02-pallet-design/h-origins/index.mdx

```diff
@@ -0,0 +1,152 @@
+---
+title: Define a custom origin as an account ID
+slug: /how-to-guides/v3/pallet-design/account-id-origin
+version: 3.0
+section: how to guides
+category: pallet design
+difficulty: 2
+---
+
+This guide goes over a simple way to assign an account ID to an origin. This origin could be used 
+from within the pallet its defined in but also in any other pallet it is tightly coupled to. This could
+be used as the basis for creating similar origin capabilities as FRAME's democracy pallet, for example 
+implementing a [special root origin](/rustdocs/latest/pallet_democracy/index.html#root).
+
+## Use cases
+
+- Create a special origin with Sudo-like access to specific dispatchables.
+- Implement a dispatchable that can only be called by the origin defined from another pallet.
+
+## Before you begin
+
+Verify the following requirements:
+
+- Read about Substrate's [origin system](/v3/runtime/origins/) to understand what they are and ways to use them.
+- You already have a working pallet and dispatchable to implement a custom origin for.
+
+## Steps
+
+1. Start by adding an Orgin type to your pallet's `Config` trait. This type will be used when you implement 
+   your pallet for your runtime:
+
+  ```rust
+  pub trait Config: frame_system::Config {
+      // -- snip -- 
+      type Origin: From<Origin>;
+      #[pallet::constant]
+      type SpecialAccountId: Get<Self::AccountId>;
+  }
+  ```
+
+  Next, we'll create the enum for our custom origin. This will be used 
+  when we implement `EnsureOrigin` on our custom origin.
+
+1. Declare the enum and specify the origin name you want to use, for example: 
+  
```

Comment:
For example, the following code declare the enum for the origin named `MyOrigins`: